ledger/daml-on-sql/src/main/scala/on/sql/Cli.scala

```diff
@@ -36,6 +36,12 @@ private[sql] final class Cli(
         config.copy(jdbcUrl = getEnv(env).orElse(
           throw new IllegalArgumentException(s"The '$env' environment variable is undefined."))))
 
+    opt[String]("sql-backend-jdbcurl")
+      .optional()
+      .text(
+        s"The JDBC connection URL to a Postgres database containing the username and password as well. If present, $name will use the database to persist its data.")
```

Comment:
@cocreature I still don't understand why the capitalization of `$name` is different here, but if you say it's all correct, I believe you.

Reply:
Ah that one is wrong ind fails to build, fixed. Basically `name` in `CommonCli`, `Name` outside.